src/envoy/utils/utils_test.cc

```diff
@@ -128,6 +128,11 @@ TEST_P(UtilsTest, GetPrincipalNoSpiffePrefix) {
   testGetPrincipal(sans, "spiffe:foo/bar", true);
 }
 
+TEST_P(UtilsTest, GetPrincipalFromSpiffePrefixSAN) {
```

Comment:
Can you also add a test when both sans are non-spiffe.

Reply:
Thanks, added